test/CMakeLists.txt

```diff
@@ -3,12 +3,14 @@ itk_module_test()
 set( MorphologicalContourInterpolationTests
   itkMorphologicalContourInterpolationTest.cxx
   )
+set(TestExe itkMorphologicalContourInterpolationTest)
 
 if(${MCI_TestWithRLEImage})
-list(APPEND MorphologicalContourInterpolationTests
-  itkMorphologicalContourInterpolationTestWithRLEImage.cxx
+  list(APPEND MorphologicalContourInterpolationTests
+    itkMorphologicalContourInterpolationTestWithRLEImage.cxx
   )
-endif(${MCI_TestWithRLEImage})
+  set(TestExe itkMorphologicalContourInterpolationTestWithRLEImage)
```

Comment:
Is it by design that you test only one of the 2 executables?

Reply:
Yes. RLEImage is an optional dependency, and I haven't figured out a good way to specify that.

Reply:
But does using RLEImage remove the support for non-RLEImage? i.e. why not test both if MCI_TestWithRLEImage is TRUE?

Reply:
You are right. I should do that!